tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -61,17 +66,25 @@ def provision(self):
         webapp_name = parameters['name']['value']
         policy_name = self.policy.data['name'].replace(' ', '-').lower()
 
-        existing_webapp = template_util.resource_exist(group_name, webapp_name)
+        existing_service_plan = self.client.app_service_plans.get(
+            group_name, parameters['servicePlanName']['value'])
 
-        if not existing_webapp:
+        if not existing_service_plan:
```

Comment:
Add a test for this logic fork?

Reply:
I added tests for the functionapp utils!